Sources/MobilePassiveData/Permissions/StandardPermission.swift

```diff
@@ -187,6 +191,8 @@ public final class StandardPermission : Permission, Codable {
             return Localization.localizedString("MOTION_PERMISSION_DENIED")
         case .photoLibrary:
             return Localization.localizedString("PHOTO_LIBRARY_PERMISSION_DENIED")
+        case .notifications:
+            return ""  // There is no perceivable reason for an assessment to require notifications. syoung 05/25/2021
```

Comment:
How about a two-part assessment where you want them to do the second part after some specified delay? NU have already indicated this is a thing they've thought of doing and our workaround of sandwiching the two parts around another assessment of the desired length wouldn't always work.

Reply:
Good point, but I can't think of a generic message for why permission would be denied.